docs/executor_swagger.md

```diff
@@ -468,6 +468,7 @@ of a single workflow step, which the executor will use as a default location to
 
 | Name | Type | Go type | Required | Default | Description | Example |
 |------|------|---------|:--------:| ------- |-------------|---------|
+| cap | string| `string` |  | | Cap is a limit on revised values of the duration parameter. If a</br>multiplication by the factor parameter would make the duration</br>exceed the cap then the duration is set to the cap |  |
```

Comment:
Can this usage example be updated to retries.md for user convenience?

Reply:
done, updated to `examples/retry-backoff.yaml`